example/bazel.repos

```diff
@@ -0,0 +1,22 @@
+repositories:
+  ogre2:
+    type: git
+    url: https://github.com/osrf/ogre-2.1-release
+    version: bazel
+  sdformat:
+    type: git
+    url: https://github.com/osrf/sdformat
+    version: bazel
+  ign_bazel:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-bazel
+    version: docker
+  ign_common:
+    type: git
+    url: https://github.com/ignitionrobotics/ign-common
+    version: fix_tests
```

Comment:
This will eventually need to be `bazel`

Reply:
I went ahead and did it, I think we can land that pr as well.